src/main/java/org/generationcp/middleware/domain/inventory/manager/LotItemDto.java

```diff
@@ -9,8 +9,14 @@
 
 	private Integer gid;
 
+	private Integer storageLocationId;
+
+	/** Ignored if storageLocationId is present */
 	private String storageLocationAbbr;
 
+	private Integer scaleId;
```

Comment:
rename to unitId

Reply:
I'll include later with the unitName refactor